src/lib/Preview.js

```diff
@@ -753,6 +754,10 @@ class Preview extends EventEmitter {
             );
         }
 
+        // Start the preview duration timer when the user starts to perceive preview's load
+        const tag = Timer.createTag(this.file.id, PREVIEW_LOAD_TIME_TAG);
```

Comment:
Do we care about this metric if `reload` is called with `skipServerUpdate = true`? That code path seems to bypass `load`

Reply:
For us, our most important metrics are going to be related to our network response times, and served content times. If it's coming from a "reload" most of the content will already be in the browser cache, which we don't care so much about.